static/tabletest.html

```diff
@@ -0,0 +1,32 @@
+<html>
+    <head>
+        <link rel="stylesheet" href="https://unpkg.com/vue-material/dist/vue-material.min.css">
+        <link rel="stylesheet" href="https://unpkg.com/vue-material/dist/theme/default.css">
+        <link href="https://unpkg.com/tabulator-tables@4.1.4/dist/css/semantic-ui/tabulator_semantic-ui.min.css" rel="stylesheet">
+
+        <style>
+            /* Tableview style */
+            #app {
+              font-family: 'Avenir', Helvetica, Arial, sans-serif;
```

Comment:
Should be in the component .vue, not in a .html.

Reply:
linked webpack css